tests/benchdnn/utils/parser.cpp

```diff
@@ -444,12 +444,25 @@ attr_t::dropout_t parse_attr_dropout_func(const std::string &s) {
     v.seed = stoll_safe(subs);
     if (start_pos == std::string::npos) return v;
 
-    v.tag = get_substr(s, start_pos, '\0');
+    v.tag = get_substr(s, start_pos, ':');
     if (check_tag(v.tag) != OK) {
         BENCHDNN_PRINT(0, "%s \'%s\' %s\n", "Error: dropout mask tag",
                 v.tag.c_str(), "is not recognized.");
         SAFE_V(FAIL);
     }
+    if (start_pos == std::string::npos) return v;
+
+    subs = get_substr(s, start_pos, ':');
+    v.offset = stoll_safe(subs);
+    if (v.offset < 0) {
+        BENCHDNN_PRINT(
+                0, "Error: bad dropout offset value: %" PRIu64 "\n", v.offset);
```

Comment:
Seems like `%lld` should be enough considering the declaration type is `int64_t`, not `uint64_t`.